starknet_py/net/models/transaction_test.py

```diff
@@ -110,7 +110,7 @@ def test_declare_compress_program(balance_contract):
                 nonce=23,
                 version=1,
             ),
-            1040693415700027927754435936502603457316156159253336216524207357712582758611,
+            1982828244129856379059233231530778257766878379971394245598825266532206245993,
```

Comment:
Why has this changed?

Reply:
I'm using different contract now

Reply:
How come? I don't see any contract changes 👽

Reply:
Uh, my bad. will fix that 👀